db/migrations/meta/_journal.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
Delete the entire migrations directory, we have automation that handle this instead